### DISCUSSION TITLE
Ferramentas para estudantes

### DISCUSSION THREAD

**Discussion by Adejair** [frontendbr/forum#159] [Discussão]
Alguns de vocês, devem estar cursando alguma faculdade ou algo do gênero, algumas empresas costumam oferecer ferramentas para desenvolvedores que são pagas, gratuitamente se você comprovar que você realmente é um estudante.

Meu intuito com este post é para reunir links de empresas que tem essas iniciativas com os estudantes, então se vocês conhecem alguma, cite aqui \o\
- [Github](https://education.github.com/pack)
- [Jetbrains](https://www.jetbrains.com/student/)

**Comment by augustoabreu** [comment]
Serviços na AWS: https://aws.amazon.com/education/awseducate/

**Comment by lagden** [comment]
@Adejair estou sempre estudando será que isso vale tb?!

**Comment by Adejair** [comment]
Então @lagden o problema é a validação, na maioria dos casos, como por exemplo, no Github e no JetBrains é necessário um ID de identificação escolar ou então algum comprovante emitido pela própia instituição comprovando que você é um aluno, acho que não é válido ;(

Normalmente se for por comprovante da instituição que é o nosso caso, já que estamos no Brasil, é sempre validado por uma pessoa física, no JetBrains eles demoraram 1 dia pra validar meu comprovante, no GitHub eles ainda não comprovaram, está a 8 dias pendente.

Então acho que você precisaria no mínimo estar matriculado em alguma instituição que emita um comprovante que você é aluno de lá para poder receber.

**Comment by wjpiantkoski** [comment]
Boa iniciativa. Vale a pena tentar o Dreamspark também, dependendo do tipo de parceria que a instituição fez, da pra conseguir acesso total ao que tem lá. As federais normalmente tem tudo liberado.

[Dreamspark](https://www.dreamspark.com/)

**Comment by evansantos** [comment]
a MS também tem esse pacote aqui: [Visual Studio Dev Essentials](https://www.visualstudio.com/pt-br/products/visual-studio-dev-essentials-vs.aspx)

**Comment by danilovaz** [comment]
Curti :+1: 

mandem seus pull requests de ferramentas aqui [tools-for-students](https://github.com/danilovaz/tools-for-students) :smiley:

**Comment by weltonsanches** [comment]
Uma ferramenta muito interessante é o [Hackerrank](https://www.hackerrank.com). Várias empresas fazem entrevistas por lá, então além de treinar o código você também já vai se habituando com o ambiente :)

**Comment by munizart** [comment]
A [Creative Cloud da Adobe](http://www.adobe.com/br/creativecloud/buy/students.html) tem um desconto de 60%

**Comment by rubenmarcus** [comment]
@WeltonSanches  nao tem nada de JS la..

**Comment by weltonsanches** [comment]
@rubenmarcus no exercício de algoritmos da pra usar várias linguagens, inclusive Javascript. Essa série é legal porque começa com desafios bem simples (o primeiro desafio é somar dois inteiros) e o nível de dificuldade vai aumentando até chegar nos algoritmos de IA classificados com NP Completo.
